lib/Service/ApiService.php

```diff
@@ -67,4 +81,35 @@ public function sendFile(int $fileId): void {
 			],
 		);
 	}
+
+	public function searchMessages(string $userId, string $term, int $offset = 0, int $limit = 10): array {
+		$arguments = [
+			'format' => 'json',
```

Comment:
I can't find anything about this format parameter in the API docs

Reply:
Well actually I am not sure about this, but if you go to /api/documents/, you can choose API or JSON. If you choose JSON, it will add "format=json" to the GET request. So, I just play safe in this case.
![image](https://github.com/user-attachments/assets/3e37b4a5-d532-4960-95bf-802d2e722d55)